_posts/2015-11-24-booting-discovery-over-http.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: 
+date: 2015-11-24 11:14:15
+author: Foreman
+tags:
```

Comment:
Do you want to add the `guest-post` and `guest-url` flags here? Your own blog deserves a link ;)